react/Icons/LogoutLarge.jsx

```diff
@@ -3,7 +3,7 @@ import React from 'react'
 
 function SvgLogoutLarge(props) {
   return (
-    <svg {...props}>
+    <svg viewBox="0 0 44 44" {...props}>
```

Comment:
We should not manually edit those files! The header says: 
```
// Automatically created, please run `scripts/generate-svg-icon.sh assets/icons/illus/logout-large.svg` to regenerate;
```

So you need to change the source file to get this viewBox!

Reply:
@doubleface